tests/integration/kubernetes/setup.sh

```diff
@@ -50,27 +50,28 @@ setup_policy_files() {
 }
 
 add_annotations_to_yaml() {
+	# Previous version of yq was not ready to handle multiple objects in a single yaml.
+	# By default was changing only the first object.
+	# With yq>4 we need to make it explicit during the read and write.
```

Comment:
nit: Maybe move this comment to just before `resource_kind` as it related to that line specifically?